src/transformers/models/timm_backbone/modeling_timm_backbone.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, config, **kwargs):
         if config.backbone is None:
             raise ValueError("backbone is not set in the config. Please set it to a timm model name.")
 
-        if config.backbone not in timm.list_models():
+        if config.backbone.split(".")[0] not in timm.list_models():
```

Comment:
This change is because certain timm checkpoints will have the base model + a bunch of specific model specifications e.g. `vit_large_patch14_dinov2.lvd142m`